packages/ui/src/components/EventsCard/EventsCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Flex, Stack } from '@chakra-ui/react';
+
+import { Box, BoxProps } from '../Box';
+import { Button } from '../Button';
+import { Link } from '../Link';
+import { Typography } from '../Typography';
+import { BUTTON_TEXT, MAIN_TITLE } from './EventsCard.mocks';
+
+export type Event = {
+  id: number;
+  title: string;
+  date: Date | { from: Date; to: Date };
+  description: string;
+  url: string;
+};
+
+export interface EventCardProps extends BoxProps {
+  events: Event[];
+}
+
+const getDayAndMonth = (date: Date): string => {
+  if (date === undefined) {
+    return '';
+  }
+
+  const month = date.getMonth() + 1;
+
+  return `${date.getDate()?.toString().padStart(2, '0')}.${month?.toString().padStart(2, '0')}`;
```

Comment:
❓ Can `getDate` return `undefined`? Also can `month` be `undefined` when the `date` isn't?